src/Analyzers/CSharp/CodeFixes/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionCodeFixProvider.Rewriter.cs

```diff
@@ -246,7 +246,7 @@ private ExpressionSyntax RewriteSwitchStatement(
             var switchStatement = topLevel ? AddCastIfNecessary(node) : node;
 
             return SwitchExpression(
-                switchStatement.Expression.Parenthesize(),
+                switchStatement.Expression.WithTrailingTrivia().Parenthesize(),
```

Comment:
This is equivalent to:

```suggestion
                switchStatement.Expression.WithoutTrailingTrivia().Parenthesize(),
```

Which reads better as to intent.

Reply:
updated